admin_manual/appliance/Backup.rst

```diff
@@ -0,0 +1,21 @@
+======
+Backup
+======
+
+If you remove the ownCloud app or update it - a backup is created automatically.
+
+The backup remains on the host system and can be restored.
+
+It is stored in :: 
+
+  /var/lib/univention-appcenter/backups/ 
+
+The file name is ::
+
+  appcenter-backup-owncloud:date
+
+There you find your data and conf folders. 
```

Comment:
s/There you find your data and conf folders./In it, you find your data and conf folders./g